### PR TITLE
portal/brand: Added settings forms

### PR DESCRIPTION
**Pull request by fmendozzac**
#### Type Of Change 
- [ ] Small bug fix
- [x] New feature or enhancement
- [ ] Breaking change

#### Checklist:
- [x] Commits are named and have tag following [commit rules](https://github.com/irontec/ivozprovider/blob/bleeding/doc/dev/en/commits.md)
- [x] Commits are split per component (schema, web/admin, kamusers, agis, ..)
- [x] Changes have been tested locally
- [ ] Fixes an existing issue (Fixes #XXXX) 
- [ ] Upport from existing Pull request #XXXX

### CODE REVIEW

**Comment by ironArt3mis** [comment]
Rebase completed.

File: web/portal/brand/src/entities/WebPortal/Form.tsx [hunk 1]
```diff
@@ -18,16 +17,12 @@ const Form = (props: EntityFormProps): JSX.Element => {
...
-        'userTheme',
-        'id',
-        'logo',
-      ],
+      legend: '',
+      fields: ['name', 'urlType', 'url'],
+    },
+    {
+      legend: '',
+      fields: ['klearTheme', 'logo'],
```
Comment (mmadariaga) [inline web/portal/brand/src/entities/WebPortal/Form.tsx:25]: klearTheme must be an enum

File: web/portal/brand/src/entities/NotificationTemplateContent/Form.tsx [hunk 2]
```diff
@@ -18,17 +18,16 @@ const Form = (props: EntityFormProps): JSX.Element => {
...
+      legend: '',
+      fields: ['language'],
+    },
+    {
+      legend: _('From configuration'),
+      fields: ['fromName', 'fromAddress'],
+    },
+    {
+      legend: _('Contents configuration'),
+      fields: ['bodyType', 'subject', 'body'],
```
Comment (mmadariaga) [inline web/portal/brand/src/entities/NotificationTemplateContent/Form.tsx:30]: body must be of type textarea

File: web/portal/brand/src/entities/MusicOnHold/Form.tsx [hunk 3]
```diff
@@ -18,8 +17,12 @@ const Form = (props: EntityFormProps): JSX.Element => {
...
   const groups: Array<FieldsetGroups | false> = [
     {
-      legend: _('Main'),
-      fields: ['name', 'status', 'id', 'originalFile', 'encodedFile'],
+      legend: '',
+      fields: ['name'],
+    },
+    {
+      legend: '',
+      fields: ['originalFile', 'encodedFile'],
```
Comment (mmadariaga) [inline web/portal/brand/src/entities/MusicOnHold/Form.tsx:25]: encodedFile must be readOnly

File: web/portal/brand/src/entities/BrandService/Form.tsx [hunk 4]
```diff
@@ -18,8 +17,12 @@ const Form = (props: EntityFormProps): JSX.Element => {
 
   const groups: Array<FieldsetGroups | false> = [
     {
-      legend: _('Main'),
-      fields: ['code', 'id', 'service'],
+      legend: '',
+      fields: ['service'],
```
Comment (mmadariaga) [inline web/portal/brand/src/entities/BrandService/Form.tsx:21]: Service must show only unassigned services and must be readOnly on edit

**Review by mmadariaga — CHANGES_REQUESTED** [review]

**Comment by ironArt3mis** [comment]
Rebase completed.

File: web/portal/brand/src/entities/WebPortal/WebPortal.tsx [hunk 5]
```diff
@@ -17,6 +17,7 @@ const properties: WebPortalProperties = {
   },
   klearTheme: {
     label: _('Theme'),
+    enum: {},
```
Comment (mmadariaga) [inline web/portal/brand/src/entities/WebPortal/WebPortal.tsx:20]: Missing values

```
    irontec-red: Irontec-red
    irontec-blue: Irontec-blue
    absolution: Absolution
    absolution-green: Absolution-green
    absolution-red: Absolution-red
    aristo: Aristo
    aristo-dark: Aristo-dark
    aristo-green: Aristo-green
    aristo-red: Aristo-red
    delta: Delta
    redmond-red: Redmond-red
    twitter: Twitter
    tedra: Tedra
    pinkpat: Pinkpat
    redinn: Redinn
    grayidec: Grayidec
    base: Base
    black-tie: Black-tie
    blitzer: Blitzer
    cupertino: Cupertino
    dark-hive: Dark-hive
    dot-luv: Dot-luv
    eggplant: Eggplant
    excite-bike: Excite-bike
    flick: Flick
    hot-sneaks: Hot-sneaks
    humanity: Humanity
    le-frog: Le-frog
    mint-choc: Mint-choc
    overcast: Overcast
    pepper-grinder: Pepper-grinder
    redmond: Redmond
    smoothness: Smoothness
    south-street: South-street
    start: Start
    sunny: Sunny
    swanky-purse: Swanky-purse
    trontastic: Trontastic
    ui-darkness: Ui-darkness
    ui-lightness: Ui-lightness
    vader: Vader
```

File: web/portal/brand/src/entities/NotificationTemplate/Form.tsx [hunk 6]
```diff
@@ -18,8 +17,8 @@ const Form = (props: EntityFormProps): JSX.Element => {
 
   const groups: Array<FieldsetGroups | false> = [
     {
-      legend: _('Main'),
-      fields: ['name', 'type', 'id'],
+      legend: '',
+      fields: ['name', 'type'],
```
Comment (mmadariaga) [inline web/portal/brand/src/entities/NotificationTemplate/Form.tsx:21]: Wrong values in type. Valid values are:

```
          'voicemail': 'Voicemail'
          'fax': 'Fax'
          'lowbalance': 'Low balance'
          'invoice': 'Invoice'
          'callCsv': 'Call CSV'
          'maxDailyUsage': 'Max daily usage'
```

File: web/portal/brand/src/entities/MusicOnHold/MusicOnHold.tsx [hunk 7]
```diff
@@ -33,6 +33,7 @@ const properties: MusicOnHoldProperties = {
   encodedFile: {
     label: _('File for play'),
     type: 'file',
+    readOnly: true,
```
Comment (mmadariaga) [inline web/portal/brand/src/entities/MusicOnHold/MusicOnHold.tsx:36]: We've an error here: "Empty download path"

**Review by mmadariaga — CHANGES_REQUESTED** [review]

**Review by mmadariaga — APPROVED** [review]

**Pull request merged**